src/viam/sessions_client.py

```diff
@@ -44,24 +46,32 @@ class SessionsClient:
 
     _current_id: str = ""
     _disabled: bool = False
-    _supported: Optional[bool] = None
     _heartbeat_interval: Optional[timedelta] = None
 
-    def __init__(self, channel: Channel, *, disabled: bool = False):
+    def __init__(self, channel: Channel, address: str, dial_options: Optional[DialOptions], *, disabled: bool = False):
         self.channel = channel
         self.client = RobotServiceStub(channel)
         self._disabled = disabled
-        self._lock = asyncio.Lock(**loop_kwargs())
+
+        self._address = address
+        self._dial_options = dial_options
+
+        self._lock: Lock = Lock()
+        self._supported: _SupportedState = _SupportedState.UNKNOWN
+        self._thread: Optional[Thread] = None
```

Comment:
It'd be nice to define these above with the other instance vars

Reply:
moved, also moved some of the class vars into this function since I don't think they should've been class vars to begin with